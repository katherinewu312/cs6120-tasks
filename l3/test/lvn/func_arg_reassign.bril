@main(in: int){
  v: int = id in;
  in: int = const 3;
  x: int = id in;
  y: int = add x v;
  print y;
}
