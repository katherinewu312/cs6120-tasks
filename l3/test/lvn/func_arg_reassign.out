@main(in: int) {
  v: int = id in;
  in: int = const 3;
  x: int = id in;
  y: int = add in v;
  print y;
}
